.gitignore

```diff
@@ -15,3 +15,4 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+.idea
```

Comment:
Editor-specific files/directories should not be part of this repo `.gitignore` file. Use global `.gitignore` on your machine instead if you're using IDEA and don't want to see its files as changed files.